test/e2e/e2e_test.go

```diff
@@ -0,0 +1,112 @@
+package e2e
+
+import (
+	"context"
+	"path/filepath"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+
+	batchv1 "k8s.io/api/batch/v1"
+	v1 "k8s.io/api/core/v1"
+
+	kueue "sigs.k8s.io/kueue/apis/kueue/v1alpha2"
+	"sigs.k8s.io/kueue/pkg/util/testing"
+)
+
+// +kubebuilder:docs-gen:collapse=Imports
+
+func CreateClientUsingCluster() (client.Client, error) {
+	var useExistingCluster = true
+	crdPath := filepath.Join("..", "..", "config", "components", "crd", "bases")
+	webhookPath := filepath.Join("..", "..", "config", "components", "webhook")
+
+	envTest := &envtest.Environment{
+		UseExistingCluster: &useExistingCluster,
+		CRDDirectoryPaths:  []string{crdPath},
+		WebhookInstallOptions: envtest.WebhookInstallOptions{
```

Comment:
Maybe we don't add this, given that they should already be installed by `make deploy`

Reply:
Removed.